src/app/__tests__/App.test.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
```

Comment:
Why has the test here been changed? Did you mean to put this in a different test file?

Reply:
It can (probably) be reset with:

```bash
git checkout origin/master -- src/app/__tests__/App.test.js
```

Let me know if that doesn't work.